bin/conjur-cli/commands/server.rb

```diff
@@ -19,14 +22,17 @@ module Commands
       file_name
       bind_address
       port
+      no_migrate
     ]
   ) do
     def call
       # Ensure the database is available
       # and the schema is up-to-date
       @migrate_database.call(
         preview: false
-      )
+      ) unless @no_migrate
+
+      @connect_database.call if @no_migrate
```

Comment:
Do we not want to keep this, regardless of the migration, to ensure the database is able to receive connections?